diesel/src/query_builder/select_statement/dsl_impls.rs

```diff
@@ -331,45 +330,44 @@ impl<F, S, D, W, O, L, Of, G, LC, LM, Modifier> ModifyLockDsl<Modifier>
             self.distinct,
             self.where_clause,
             self.order,
-            self.limit,
-            self.offset,
+            self.limit_offset,
             self.group_by,
             LockingClause::new(self.locking.lock_mode, modifier),
         )
     }
 }
 
-impl<'a, F, S, D, W, O, L, Of, G, DB> BoxedDsl<'a, DB>
-    for SelectStatement<F, SelectClause<S>, D, W, O, L, Of, G>
+impl<'a, F, S, D, W, O, LOf, G, DB> BoxedDsl<'a, DB>
+    for SelectStatement<F, SelectClause<S>, D, W, O, LOf, G>
 where
     Self: AsQuery,
     DB: Backend,
     S: QueryFragment<DB> + SelectableExpression<F> + 'a,
     D: QueryFragment<DB> + 'a,
     W: Into<BoxedWhereClause<'a, DB>>,
     O: Into<Option<Box<QueryFragment<DB> + 'a>>>,
-    L: QueryFragment<DB> + 'a,
-    Of: QueryFragment<DB> + 'a,
+    LOf: QueryFragment<DB> + 'a,
     G: QueryFragment<DB> + 'a,
 {
     type Output = BoxedSelectStatement<'a, S::SqlType, F, DB>;
 
     fn internal_into_boxed(self) -> Self::Output {
-        BoxedSelectStatement::new(
-            Box::new(self.select.0),
-            self.from,
-            Box::new(self.distinct),
-            self.where_clause.into(),
-            self.order.into(),
-            Box::new(self.limit),
-            Box::new(self.offset),
-            Box::new(self.group_by),
-        )
+        unimplemented!()
```

Comment:
I've just want to get the normal `SelectStatement` working before I will fix that.